src/app/webdav/blob-service.service.ts

```diff
@@ -0,0 +1,131 @@
+import { Injectable } from '@angular/core';
+import { AppConfig } from '../app-config/app-config';
+import webdav from 'webdav';
+import { DomSanitizer, SafeUrl } from '@angular/platform-browser';
+import { SessionService } from 'app/session/session.service';
+
+@Injectable()
+export class BlobService {
```

Comment:
`BlobService` isn't making it obvious what this does. Now the service seems tightly related to webdav and saving/loading images, maybe it could be `WebdavFileService` or `CloudFileService` or something similar?

Reply:
I will rename it.